### DISCUSSION TITLE
is there a way to specify the output file time scale

### DISCUSSION THREAD

**Discussion by solosue** [CH-Earth/summa#491] [Q&A] [answered]
#### Bug Reports
I'm using SUMMA Version 3.0.4. 
I might be wrong, but I didn't find a way to specify the output file time period (e.g. output in every month or every year). It seems that I can only get two output files (run1_timestep.nc and run1_day.nc). If the run fails before it finish, I got empty output files.
Thank you for your help.

**Comment by wknoben** [comment, marked as answer]
Hi, 
Assuming a run completes successfully, you can control which variables go into the output file and at what temporal resolution by modifying your `outputControl.txt` file. This is the file that contains lines such as:

```
scalarSWE | 1
```

The first column indicates the variable (you can look into the file `var_lookup.f90` to see which variables are available: https://github.com/NCAR/summa/blob/master/build/source/dshare/var_lookup.f90). The second column indicates the number of timesteps over which to aggregate the simulation. For example, `outputControl.txt` for a case where you have hourly simulations but want daily outputs would look as follows:

```
scalarSWE | 24
```

The SUMMA docs provide more details: https://summa.readthedocs.io/en/latest/input_output/SUMMA_input/#output-control-file
